edi_oca/tests/test_record.py

```diff
@@ -42,9 +42,7 @@ def test_record_validate_state(self):
                 "edi_exchange_state": "output_pending",
             }
             self.backend.create_record("test_csv_input", vals)
-            self.assertEqual(
-                err.exception.name, "Exchange state must respect direction!"
-            )
+        self.assertEqual(err.exception.name, "Exchange state must respect direction!")
```

Comment:
It would be probably better to use `assertRaisesRegex` instead

Reply:
Actually is not. Something else in the stack could throw that same type of exception and you won't notice it.
Now, ideally, we should have a specific exception instead of this generic one... then you'll be 100% right :wink: